src/rtf-plugins/fixture-managers/yarpplugin/YarpPluginFixture.cpp

```diff
@@ -42,11 +52,11 @@ bool YarpPluginFixture::setup(int argc, char** argv) {
     if(argc<=1)
     {
         //RTF::RTF_FIXTURE_REPORT();
-        RTF_ASSERT_ERROR("YarpPluginFixture: Please specify --devices and/or --plugins and/or --carriers parameters");
+        RTF_ASSERT_ERROR("YarpPluginFixture: Please specify --devices and/or --portmonitor and/or --carriers parameters");
```

Comment:
`--plugins` is missing here.

Also `--devices`, `--carriers` and `--plugins` are plural, i think this should be plural as well...